src/exercises/control-flow-pointer/answers.md

```diff
@@ -27,3 +27,21 @@ ca1            0xd11720000801800600000000aaaaaaaa       0xaaaaaaaa [rxR,0xaaaa00
 => 0x101c5e <main+58>:      cjalr   cra,ca1
 ```
 The program attempted to load an instruction via an untagged capability `ca1`.
+
+# Answers - Corrupt a control-flow pointer using a subobject buffer overflow
+
+2. Example session:
+```
+(gdb) r
+Starting program: /root/buffer-overflow-fnptr-morello 
+
+Program received signal SIGSEGV, Segmentation fault.
```

Comment:
This exercise needs modifying so it doesn't perform a misaligned access, which (bogusly) takes higher precedence on Morello (making it look like the capability check has been bypassed)